### PR TITLE
🔨 Data pages: ignore indicator level grapher configs on /grapher/ pages for now

### PR DESCRIPTION
**Pull request by danyx23**
This PR changes how grapher configs are treated in data pages. Grapher configs coming from the ETL level grapher config are no longer taken into account on /grapher/ pages, only in pure datapage previews.

The PR also streamlines the code between previews and baking a bit.

Once our grapher admin can deal with defaults in a better way (i.e. show the user where defaults are used and given them an option to disable the default value) we should change this again.

### CODE REVIEW

**Comment by danyx23** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2957** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2957?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #2958** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/2958?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2957?utm_source=stack-comment).

File: baker/GrapherBaker.tsx [hunk 1]
```diff
@@ -64,12 +64,9 @@ import { Gdoc } from "../db/model/Gdoc/Gdoc.js"
...
 import { isEmpty } from "lodash"
 
-/**
- *
- * Render a datapage if available, otherwise render a grapher page.
- */
-export const renderDataPageOrGrapherPage = async (
+const renderDatapageIfApplicable = async (
     grapher: GrapherInterface,
+    isPreviewing: boolean,
```
Comment (sophiamersmann) [inline baker/GrapherBaker.tsx:69]: this is currently not used, I think?

**Review by sophiamersmann — APPROVED** [review]
LGTM!

**Comment by danyx23** [comment]
### Merge activity

* **Nov 28, 9:54 AM**: @danyx23 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/2957).

**Pull request merged**